tests/test_element.py

```diff
@@ -236,3 +236,11 @@ def test_points(SR, N):
     elem.addBluePrint(1, bp)
 
     assert elem.points == N
+
+
+def test_write_read_element(blueprint_tophat):
+    elem = Element()
+    elem.addBluePrint(1, blueprint_tophat)
+    elem.write_to_json('testfile.json')
```

Comment:
It's a good idea to use a temporary directory for testing, so that running the tests does not create additional files on the user's machine. See here: https://docs.python.org/3/library/tempfile.html and e.g. this test: https://github.com/QCoDeS/Qcodes/blob/d8d359dc6c7e2e7cab3d5a8bc355cd96b63621a8/qcodes/tests/test_config.py#L142 from qcodes

Reply:
It is even better to use the tmp_path fixture from pytest (which is what we do in qcodes mostly) https://docs.pytest.org/en/stable/tmpdir.html

Reply:
the test code now is now using tmp_path